config.yaml

```diff
@@ -15,7 +15,7 @@ options:
     description: "the length of a single tick, which is the basic time unit used by ZooKeeper, as measured in milliseconds."
     type: int
     default: 2000
-  log_level:
+  log-level:
```

Comment:
The code-convention would say that we should use `_` (see [here](https://juju.is/docs/sdk/styleguide)) for config options and actions, but I see a lot of charms using `-`. 

I'll bring this up on a charming engineering sync tomorrow and report back what it is the way forward, whether we should change the other options or log-level.

Reply:
We have discussed this on the engineering sync and we had decided that it is more natural to nudge people towards `enable-feature` convention. In fact, if you go in the link above, the description should have now been rephrased to clarify where we stand.